src/chocolatey.tests/infrastructure.app/utility/ArgumentsUtilitySpecs.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace chocolatey.tests.infrastructure.app.utility
+{
+    using chocolatey.infrastructure.app.utility;
+    using NUnit.Framework;
+    using Should;
+
+    public class ArgumentsUtilitySpecs
+    {
+        public abstract class ArgumentsUtilitySpecsBase : TinySpec
+        {
+            public override void Context()
+            {
+            }
+        }
+
+        [TestFixture("choco install bob --package-parameters-sensitive=\"/test=bill\"", true)]
+        [TestFixture("choco install bob -package-parameters-sensitive=\"/test=bill\"", true)]
+        [TestFixture("choco install bob --install-arguments-sensitive=\"/test=bill\"", true)]
+        [TestFixture("choco install bob -install-arguments-sensitive=\"/test=bill\"", true)]

```

Comment:
Should maybe test for the same with `upgrade` and `uninstall`? (I know parameters can be used in `uninstall`, but unsure if install arguments can though).